portfolio/src/main/java/com/google/sps/servlets/HardStringServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.swing.text.Document;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/hardString")
+public class HardStringServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> array = new ArrayList<>();
```

Comment:
Instead of ArrayList you can use a Array allowing you to initialize in a single line, vs multiple add statements.

Reply:
Will try to implement that, thanks for the suggestion!